sections/GPU.tex

```diff
@@ -1,14 +1,21 @@
-\section{Graphics Processing Units}
-\label{sec:GPU}
-Dedicated Graphic Processing Units - GPUs - are very common in desktop and laptops. \citep{STEAMHW}
+\section{Computation architectures}
+\label{sec:comparch}
+Normally computations are performed on a computer's Central Processing Unit - a CPU.
+A CPU is often fastest for everyday task and calculations.
```

Comment:
Everyday "tasks" - jeg bruger min GPU ret ofte i forhold til spil, hvordan er det ikke en everyday task?
A CPU is often fastest for calculations and task not requiring immense computing power, as well as any calculation not parallelisable